### ISSUE TITLE
CMake has some warnings we should take care of

### ISSUE THREAD

**Issue by royfalk** [vegastrike/Vega-Strike-Engine-Source#934]
CMP0102 and CMP0107. We should really do a review of CMake output and see what else needs fixing, in order to reduce technical debt.

**Comment by BenjamenMeyer** [comment]
Yes we probably do; but let's do this on master after my CMake refactor has been merged. (#889)
